From 0a16ae1635693e9423d67164c6526c129ebea9c5 Mon Sep 17 00:00:00 2001
From: David Gibson <dgibson@redhat.com>
Date: Mon, 15 Sep 2014 07:13:30 -0400
Subject: [virt] kvm/ppc: book3s - Move little endian conflict to HV KVM

Message-id: <1410765214-16377-75-git-send-email-dgibson@redhat.com>
Patchwork-id: 94747
O-Subject: [PATCH 74/78] KVM: PPC: Book3S: Move little endian conflict to HV KVM
Bugzilla: 1123145 1123133 1123367
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Signed-off-by: David Gibson <dgibson@redhat.com>

commit f24bc1ed45c13805c19f58cc4289b1be7efee19b
Author: Alexander Graf <agraf@suse.de>
Date:   Thu Apr 24 13:55:36 2014 +0200

    KVM: PPC: Book3S: Move little endian conflict to HV KVM

    With the previous patches applied, we can now successfully use PR KVM on
    little endian hosts which means we can now allow users to select it.

    However, HV KVM still needs some work, so let's keep the kconfig conflict
    on that one.

    Signed-off-by: Alexander Graf <agraf@suse.de>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kvm/Kconfig b/arch/powerpc/kvm/Kconfig
index b1ab43f..296b181 100644
--- a/arch/powerpc/kvm/Kconfig
+++ b/arch/powerpc/kvm/Kconfig
@@ -6,7 +6,6 @@ source "virt/kvm/Kconfig"
 
 menuconfig VIRTUALIZATION
  bool "Virtualization"
- depends on !CPU_LITTLE_ENDIAN
  ---help---
    Say Y here to get to see options for using your Linux host to run
    other operating systems inside virtual machines (guests).
@@ -76,6 +75,7 @@ config KVM_BOOK3S_64
 config KVM_BOOK3S_64_HV
  tristate "KVM support for POWER7 and PPC970 using hypervisor mode in host"
  depends on KVM_BOOK3S_64
+ depends on !CPU_LITTLE_ENDIAN
  select KVM_BOOK3S_HV_POSSIBLE
  select MMU_NOTIFIER
  select CMA
-- 
1.7.1